core/common-ui/src/main/java/team/noweekend/core/common/ui/calendar/util/CalendarUtils.kt

```diff
@@ -7,9 +7,14 @@ import kotlinx.datetime.Month
 import kotlinx.datetime.minus
 import kotlinx.datetime.number
 import kotlinx.datetime.plus
+import team.noweekend.core.common.kotlin.extension.instant
+import kotlinx.datetime.TimeZone
 
 object CalendarUtils {
 
+
+    val currentTime = instant.toLocalDateTime(TimeZone.currentSystemDefault())
```

Comment:
p3 : 이부분 날짜 관련이라 자주사용될 것 같은데
LocalDateTime.kt 파일 하나 만들어서 아래처럼 쓰는거 어때
```suggestion
    fun LocalDateTime.Companion.now(): LocalDateTime = instant.toLocalDateTime()
```

만들어놓은 Instant 파일 보면 기본 TimeZone따라서 LocalDateTime으로 변환하는 코드는 넣어놨어

Reply:
@JaesungLeee 
그 만들어둔거 있어서 
그냥 새로운 변수를 하나 할당했어

Reply:
LocalDate.now() 요걸로 하면 기존에 만들어둔거 쓸 수 있을텐데?!?

Reply:
@JaesungLeee 
이것도 만들어 있는 util을 사용한거라